### PR TITLE
Add CHX collection metapkg and require 'chxanalys' package for analysis.

### PR DESCRIPTION
**Pull request by danielballan**
@yugangzhang Please confirm that these two metapackages includes everything you need for collection and analysis, respectively.

### CODE REVIEW

File: recipes-tag/11-id-chx-analysis/meta.yaml [hunk 1]
```diff
@@ -7,10 +7,11 @@ package:
...
 build:
-  number: 0
+  number: 1
 
 requirements:
   run:
     - chxtools
     - eiger-io
     - hdf5-lz4
+    - chxanalys
```
Comment (yugangzhang) [inline recipes-tag/11-id-chx-analysis/meta.yaml]: Add
dask
reportlab
theano
maya
graphviz
boto3

Comment (danielballan) [inline recipes-tag/11-id-chx-analysis/meta.yaml]: @licode CHX needs these as well. Most are probably available from anaconda. Can you identify which (if any) need to be copied from conda-forge?

Comment (arkilic) [inline recipes-tag/11-id-chx-analysis/meta.yaml]: conda-forge: dask, theano, graphviz, boto3
defaults: reportlab

Is [this](https://github.com/kennethreitz/maya) the maya @yugangzhang needs?

**Comment by danielballan** [comment]
Are you sure official conda does not package dask?

Get Outlook for iOS<https://aka.ms/o0ukef>




On Thu, Jan 12, 2017 at 2:25 PM -0500, "Arman Arkilic" <notifications@github.com<mailto:notifications@github.com>> wrote:


@arkilic commented on this pull request.

________________________________

In recipes-tag/11-id-chx-analysis/meta.yaml<https://github.com/NSLS-II/lightsource2-recipes/pull/89>:

>
 requirements:
   run:
     - chxtools
     - eiger-io
     - hdf5-lz4
+    - chxanalys


conda-forge: dask, theano, graphviz, boto3
defaults: reportlab

Is this<https://github.com/kennethreitz/maya> the maya @yugangzhang<https://github.com/yugangzhang> needs?

**Comment by arkilic** [comment]
@danielballan yes, they do. All packages that are available via defaults and conda-forge (dask, theano, graphviz, boto3, reportlab) with the exception of maya. Sorry about the confusion.

**Comment by yugangzhang** [comment]
@arkilic yes, we need maya for the data analysis. Thanks!

File: recipes-tag/11-id-chx-analysis/meta.yaml (hunk 1 above)
Comment (yugangzhang) [inline recipes-tag/11-id-chx-analysis/meta.yaml]: @arkilic yes, that's the github I download maya.

**Comment by danielballan** [comment]
What's the status of this packaging work? We postponed Maya -- what about the rest?

**Comment by danielballan** [comment]
Merging because this is needed at CHX now.

**Pull request merged**